cypress/fixtures/zeroPetApplications.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Nice! Two different fixtures to account for the conditional rendering- if there aren't any applications then a p tag will tell the user what's going on. If there are (ie the petApplicationsPage fixture) then the applications will be displayed.